src/main/java/MessagePrinter.java

```diff
@@ -0,0 +1,33 @@
+public class MessagePrinter {
+    public MessagePrinter() {
+    }
+    public static void showWelcomeMessage() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
Perhaps refactor this into a constant to avoid using magic literals? 

https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers